enhancements/oc/mirroring-manifests.md

```diff
@@ -105,78 +98,104 @@ mirror
+Assuming the aforementioned use case and the use of removable media that will be sneakernetted across the gap, the steps for mirroring are:
+
+1. Connect the removable media to a system connected to the internet.
+2. Mirror the images and manifests to a directory on the removable media:
+	```console
+	$ oc adm release mirror --to-dir=<removable-media-path>/mirror quay.io/openshift-release-dev/ocp-release:4.3.0-x86_64
+	```
+3. The enhanced `oc adm release mirror ...` command will output the path, similar to the example below, to a configmap created containing the image signature. Note this path as it will be used later when the configmap is applied to a cluster to be upgraded. 
+	```console
+	Configmap <removable-media-path>/mirror/manifests/sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92 created
+	...
+4. Run any required checks/scrubs on the removable media.
+5. Connect the removable media to a host within the air gapped environment that has access to both the central registry and to any cluster requiring upgrade.
+6. Upload the mirrored images to the central registry:
+	```console
+	$ oc image mirror --from-dir=<removable-media-path>/mirror file://openshift/release:4.3.0* REGISTRY/REPOSITORY
+	```
+7. Use oc to login to a given cluster to be upgraded.
+8. Apply the the mirrored configmap, noted above, to the connected cluster for use by CVO for upgrade image verification:
+	```console
+	$ oc apply -f <removable-media-path>/mirror/manifests/sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+	```
+9. Perform the upgrade:
+	```console
+	$ oc adm upgrade --to-image REGISTRY/REPOSITORY/release@sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+	```
 
 ### User Stories
 
 #### Release-signature ConfigMaps
 
-FIXME
+Currently a cluster upgrade can be accomplished on a cluster that does not have an active connection to the internet.
+However manual steps are required to create a ConfigMap containing the signature data required for update image verification.
+This enhancement will automatically create the ConfigMap from the image manifest files.
 
 #### In-cluster Cincinnati
 
 FIXME
 
 ### Risks and Mitigations
 
-FIXME
+One risk considered is that a user familiar with the current behavior of the `oc adm release mirror ...` command may inadvertently apply manifests to thier cluster.
+This has been addressed by requiring the explicit `--apply-manifests` argument to apply manifests to a cluster.
 
 ## Design Details
 
 ### Test Plan
 
-FIXME
+The current e2e test, that makes sure the command always exits successfully and that certain apsects of the content are always present, will be modified to validate the new functioanlity as well.
 
 ### Graduation Criteria
 
-FIXME
-
 ### Upgrade / Downgrade Strategy
 
-FIXME
+The image is included in the payload, but has no content running in a cluster to upgrade.
 
 ### Version Skew Strategy
```

Comment:
This section should maybe grow some text about what to do if vA `oc` is used to mirror into a vB cluster.  For example, 4.5 `oc` (or whenever this lands) will be able to mirror signature ConfigMaps into the cluster.  But what if it's a 4.2 cluster that ignores those ConfigMaps?  Should the `oc adm release mirror ...` command check for that and error out?  Silently succeed and hope the user can figure out why their 4.2 CVO is still unable to find the signature?  What if we add some new manifest type getting mirrored in the 4.6 `oc`, and the user attempts to apply it to a 4.5 cluster that doesn't understand the new-in-4.6 type?

Reply:
+1, We need to address
```
* user uses 4.2 `oc` binary to run copy images `oc adm release mirror --to-dir=` to a directory
* updates to 4.5 `oc` binary and runs the --apply-manifests command. So CVO does not get the signature manifests 
```

With respect to  future version where we have a new manifest , we need to update oc command to check if the manifest is present and error out accordingly. That's why the structure of manifest directory is important as it should have room for future  enhancements.